modules/update-mco-process.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * updating/understanding_updates/how-updates-work.adoc
+
+:_content-type: CONCEPT
+[id="mco-update-process_{context}"]
+= Machine Config Operator node updates
+The Machine Config Operator (MCO) applies a new machine configuration to each control plane and compute node. During this process, the MCO performs the following sequential actions in the cluster:
+
+. Cordon and drain all of the nodes
+
+. Update the operating system (OS) of each node
+
+. Reboot the nodes
+
+. Uncordon all of the nodes
+
+. Schedule workloads on the nodes
+
+[NOTE]
+====
+When a node is cordoned, workloads cannot be scheduled to it.
+====
+
+The time to complete this process depends on several factors including the node and infrastructure configuration. This process might take 5 or more minutes to complete per node.
```

Comment:
I think there is deeper context to this from when this content was first made for the "Understanding update duration" page. 

@LalatenduMohanty, @petr-muller, or @wking might know, but my memory/understanding is that it's intentionally omitted to communicate that: "This process may take a long time but there are too many variations and factors for us to guarantee a specific timeframe this would be completed by". I could be wrong though

Reply:
It's very dependent on circumstances - user workloads specifically. Your understanding is correct, it's hard to guarantee. It is also possible to configure the workloads in a way that completely stucks the upgrade, unfortunately.

Reply:
hmm.. then it make sense to omit since we cannot guarantee... (unless there's some high mark that we **can** actually consider as: ok it took way too much time at this point, its probably stuck and not just taking longer than ususal)

Reply:
The machine-config operator will fire a `warning` [`MCDDrainError`][1] [after][2] [an hour][3].  But :shrug:, many workloads like the Prow CI jobs we run as CVO presubmits take multiple hours in unevictable, PodDisruptionBudget-protected pods, and that's normal too.  I would rather not get opinionated about how long "surprisingly long" is, and leave that up to the cluster administrator and workload administrators to negotiate between themselves.

[1]: https://github.com/openshift/machine-config-operator/blob/5843b7a4b27f15bc3313ce8a41ef6de792081fcb/install/0000_90_machine-config-operator_01_prometheus-rules.yaml#L20
[2]: https://github.com/openshift/machine-config-operator/blob/5843b7a4b27f15bc3313ce8a41ef6de792081fcb/pkg/controller/drain/drain_controller.go#L351
[3]: https://github.com/openshift/machine-config-operator/blob/5843b7a4b27f15bc3313ce8a41ef6de792081fcb/pkg/controller/drain/drain_controller.go#L70